src/caffe/proto/caffe.proto

```diff
@@ -1231,4 +1231,11 @@ message PReLUParameter {
   optional FillerParameter filler = 1;
   // Whether or not slope paramters are shared across channels.
   optional bool channel_shared = 2 [default = false];
+  // Whether or not to constrain the negative slope between min_neg_slope and
+  // max_neg_slope.
+  optional bool constrain_neg_slope = 3 [default = true];
```

Comment:
Hmm, yeah.  I agree that this is a reasonable updated default.  It does change experiments that people may want to re-run, but caffe has already changed training defaults in the past (https://github.com/BVLC/caffe/pull/2321).